CONTRIBUTING.md

```diff
@@ -2,11 +2,17 @@
 
 ## **Did you find a bug?**
 
+Please file an issue [here](https://github.com/caos/zitadel/issues/new/choose).
+
 ## **Want to contribute code?**
 
-* Check out our [Dev Build Guide](build/README.md).
+### build for local development
```

Comment:
Can we get a more self explaining title here?

I don't get the difference to the `docker` option below

Reply:
I updated the titles